themes-default/slim/views/vue-components/name-pattern.mako

```diff
@@ -0,0 +1,378 @@
+<script type="text/x-template" id="name-pattern-tempate">
+    <div id="name_pattern_wrapper">
+
+        <div v-if="namingCustom" class="field-pair">
+            <input type="checkbox" class="enabler" id="enable_naming_custom" name="enable_naming_custom" v-model="isEnabled"/>
+            <label for="enable_naming_custom">
+                <span class="component-title">Custom {{ namingCustom }}</span>
+                <span class="component-desc">Name {{ namingCustom }} shows differently than regular shows?</span>
+            </label>
+        </div>
+
+        <div v-if="!namingCustom || isEnabled" class="episode-naming">
+                <div class="field-pair">
+                        <label class="nocheck" for="name_presets">
+                            <span class="component-title">Name Pattern:</span>
+                            <span class="component-desc">
+                                <select id="name_presets" class="form-control input-sm" v-model="selectedNamingPattern" @change="updatePatternSamples">
+                                    <option id="preset" v-for="preset in namingPresets">{{ preset }}</option>
+                                </select>
+                            </span>
+                        </label>
+                    </div>
+                    <div id="naming_custom">
+                        <div class="field-pair" style="padding-top: 0;">
+                            <label class="nocheck">
+                                <span class="component-title">
+                                    &nbsp;
+                                </span>
+                                <span class="component-desc">
+                                    <input v-if="isCustom" type="text" name="naming_pattern" id="naming_pattern" v-model="customName" @change="updatePatternSamples" class="form-control input-sm input350"/>
+                                    <img src="images/legend16.png" width="16" height="16" alt="[Toggle Key]" id="show_naming_key" title="Toggle Naming Legend" class="legend" @click="showLegend = !showLegend" />
+                                </span>
+                            </label>
+                        </div>
+                        <div id="naming_key" class="nocheck" v-if="showLegend">
+                                <table class="Key">
```

Comment:
This should be using 2 spaces like the rest of the HTML.

Reply:
Ive used 4 spaces for all now.

Reply:
Ah, this is `        ` 8 I think, double indented?